code/modules/holiday/halloween/halloween.dm

```diff
@@ -176,7 +176,7 @@
 		return
 
 /mob/living/simple_animal/shade/howling_ghost/adjustHealth()
-	. = 0
+	. = ..() 
```

Comment:
You can just remove this then, all it does is call parent, which it kinda does by default.